src/frame/endpoints/GroupApiEndpoints.ts

```diff
@@ -142,6 +143,7 @@ function addAdmins(groupID: string, addedAdmins: EncryptedAccessKey[]) {
                     ...admin.encryptedPlaintext,
                 })),
             }),
+            sugnature: fromByteArray(signature),
```

Comment:
sugnature? Also you should manually test that this request will fail if the signature is invalid. This is currently being sent to the server as an invalid name (thus all the signing work you did was completely being ignored by ironcore-id).

I don't know if you can actually write a test on this? I'm pretty sure you can. @ernieturner would be more helpful on that than I will, but if he's not around I'm happy to help.

Reply:
@coltfred Do you mean writing an integration test that actually tries to hit ironcore-id with an invalid signature? Or are you thinking of something else?

Reply:
No, I thought you had tests that validated the structure that was passed to `fetch`. That test ends up looking pretty dumb as you just assert that the same structure you just produced is sent.

The best test I have for this in ironweb is to send an invalid signature and verify that it fails from the test app and then fix it and verify it succeeds. We did something similar in ironoxide.

Reply:
Right, [my comment below](https://github.com/IronCoreLabs/ironweb/pull/58/files#r343837747) addressed that problem here that we weren't asserting that the signature field was part of the call to fetch.

Reply:
Fixed these issues. I also tested locally that if I call addAdmins with an invalid signature the action will fail.